src/importexport/bww/internal/bww/symbols.cpp

```diff
@@ -29,7 +29,7 @@
 #include "symbols.h"
 
 namespace Bww {
-static const char* symTable[] =
+static constexpr const char* symTable[] =
```

Comment:
isnt't that `const` now redundant?

Reply:
Hmm, apparently not, not in C11 at least

Reply:
The `const` applies to the `char`, and the `const` implied by `constexpr` applies to the `*`. You can't make a `constexpr char*`, but you can make a `constexpr const char*`.